cypress/e2e/dashboard_spec.cy.ts

```diff
@@ -1,122 +1,95 @@
-})
-it('Should have a info button on the featured dog', () => {
-  cy.intercept('GET', 'https://api.thedogapi.com/v1/breeds', { body: [{
-        weight: {
-        imperial: "65 - 100",
-        metric: "29 - 45"
-        },
-        height: {
-        imperial: "23 - 25",
-        metric: "58 - 64"
         },
-        id: 9,
-        name: "Alaskan Malamute",
-        bred_for: "Hauling heavy freight, Sled pulling",
-        breed_group: "Working",
-        life_span: "12 - 15 years",
-        temperament: "Friendly, Affectionate, Devoted, Loyal, Dignified, Playful",
-        reference_image_id: "dW5UucTIW",
-        image: {
-        id: "dW5UucTIW",
-        width: 1023,
-        height: 769,
-        url: "https://cdn2.thedogapi.com/images/dW5UucTIW.jpg"
-        },
-    }]
-  }).as('random-featured-dog')
-  cy.visit('http://localhost:3000')
-  cy.get('.featured-dogs-container > :nth-child(1)')
-  cy.wait('@random-featured-dog')
-  .get('img[class="info-icon"]')
-  .click().url().should('include', 'http://localhost:3000/Alaskan%20Malamute')
-})
-})
\ No newline at end of file
+      ],
+    }).as("random-featured-dog");
+    cy.visit("http://localhost:3000");
+  });
+
+  it("should show random featured dogs", () => {
+    cy.get(".featured-dogs-container > :nth-child(1)");
+    cy.wait("@random-featured-dog").get("h3").contains("Alaskan Malamute");
+  });
+  it("should have a favorite button on the featured dog that adds the dog to favorites", () => {
+    cy.get(".featured-dogs-container > :nth-child(1)");
+    cy.wait("@random-featured-dog")
+      .get('img[class="favorite-image"]').should('be.visible')
+  });
+  it("should have a info button on the dog card that once clicked takes user to dog info page", () => {
+    cy.get(".featured-dogs-container > :nth-child(1)");
```

Comment:
is the format just being changed in this?

Reply:
Yeah, I believe I just made all the "shoulds" lower case to be consistent with the others.